apps/cotersus-e2e/project.json

```diff
@@ -1,12 +1,11 @@
 {
   "name": "cotersus-e2e",
   "$schema": "../../node_modules/nx/schemas/project-schema.json",
-  "root": "apps/cotersus-e2e",
```

Comment:
this removal is causing the ci to fail because it cannot determine the affected projects
I'm trying to fix it through [this issue](https://github.com/robinpellegrims/pellegrims/issues/216)